.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
+env:
+  - PYTHON_VERSION=2.7
+before_install:
+  - wget http://repo.continuum.io/miniconda/Miniconda3-3.7.3-Linux-x86_64.sh -O miniconda.sh
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b
+  - export PATH=/home/travis/miniconda3/bin:$PATH
+  # Update conda itself
+  - conda update --yes conda
+install:
+  - conda create --yes -n env_name python=$PYTHON_VERSION pip nose numpy scipy matplotlib pandas>=0.15 flake8
+  - conda install --yes -n env_name python=$PYTHON_VERSION h5py>=2.3.1 statsmodels lxml
+  - source activate env_name
+  # upgrade click, older versions of click do not have __version__
+  - pip install -U click
+  - pip install scikit-bio
+  - pip install biom-format
+  - pip install colorbrewer
+script:
+  - nosetests --with-doctest
+  - flake8 americangut/*.py
```

Comment:
Ah ok, I don't think they *have* to be together, but it seems strange
that they aren't. It's up to you, and there's no "travis.yml" guidelines
so yeah ... 

On (Mar-26-15|17:32), adamrp wrote:
> > +  - ./miniconda.sh -b
> > +  - export PATH=/home/travis/miniconda3/bin:$PATH
> > +  # Update conda itself
> > +  - conda update --yes conda
> > +install:
> > +  - conda create --yes -n env_name python=$PYTHON_VERSION pip nose numpy scipy matplotlib pandas>=0.15 flake8
> > +  - conda install --yes -n env_name python=$PYTHON_VERSION h5py>=2.3.1 statsmodels lxml
> > +  - source activate env_name
> > +  # upgrade click, older versions of click do not have __version__
> > +  - pip install -U click
> > +  - pip install scikit-bio
> > +  - pip install biom-format
> > +  - pip install colorbrewer
> > +script:
> > +  - nosetests --with-doctest
> > +  - flake8 americangut/*.py
> 
> I actually found it helpful during debugging to have these separated, so that all of the flake8 failures pertaining to the scripts were lumped together and all the failures related to the library code was lumped together. But if they *should* be put together for some other reason, I can do that.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/American-Gut/pull/139/files#r27269087